setup.py

```diff
@@ -22,10 +22,11 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 from setuptools import setup
+from s3po import __version__
```

Comment:
If you do this, it's important that this file contain no other imports so that the `setup.py` can't fail prior to installation, and the current `__init__.py` does contain some other imports.

I'd recommend moving this single constant to a separate file say `s3po/version.py` and do the import from there.